astroquery/vo_service_discovery/tests/shared_registry.py

```diff
@@ -0,0 +1,169 @@
+import os
+from astropy.table import Table
+from astroquery.vo_service_discovery import Registry
+
+
+class Helpers(object):
+    ""
+
+    DATA_FILES = {}
+
+    def table2ecsv(self, current, filepath):
+        """Dump a table and its meta data to ECSV"""
+        from astropy.io import ascii as ap_ascii
+        try:
+            ap_ascii.write(current, filepath, format='ecsv', overwrite=True)
```

Comment:
I don't think we need to import `io.ascii` here, the Table writer should be able to deal with it and it's already been imported.

Reply:
Good catch.  I removed the import and used the instance write() method for the incoming Table.